scripts/mesoscale/plots/exevs_mesoscale_atmos_sref_cape_plots.sh

```diff
@@ -241,6 +240,10 @@ done
 
 tar -cvf evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar *.png
 
+if [ $SENDDBN = YES ] ; then
+     $DBNROOT/bin/dbn_alert MODEL EVS_RZDM $job $COMOUT/evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar
```

Comment:
@BinbinZhou-NOAA I'm not sure that this is the correct path to your .tar file (based on the path in the SENDCOM block. These paths should point to the same place. Should this path be: 

$COMOUT/$STEP/$COMPONENT/$RUN.$VDATE/evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar

Tagging @PerryShafran-NOAA and @ShelleyMelchior-NOAA so that they check this here and in other places. The path in SENDDBN should be the path it gets moved to in the SENDCOM block.

Reply:
This is a good catch, Alicia.